cmd/thv-operator/controllers/virtualmcpserver_vmcpconfig.go

```diff
@@ -0,0 +1,527 @@
+package controllers
```

Comment:
I'd put this into it's own package. I was half way through doing the runconfig, beausee the controllers package has too much code in it that breaks single responsibility